static/chats.js

```diff
@@ -156,16 +165,51 @@ function saprotiKomandu(teksts) {
 }
 
 function uzstadiVaardu(jaunaisVards) {
-  const vecaisVards = vards
-  setCookie('name', jaunaisVards, 90)
-  vards = jaunaisVards
+  const vecaisVards = vards;
+  vards = jaunaisVards;
   return `${vecaisVards} kļuva par ${vards}`
 }
 
+function nolasiGarastavokli(gStavoklis){
+  const gStavoklisUzServeri = new GaraStavoklis(vards, gStavoklis.join(' '))
+  let parameters = {
+    method: 'POST',
+    body: JSON.stringify({"mood": gStavoklisUzServeri}),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }
+  fetch('/garastavoklis/pierakstit_grarastavokli', parameters)
+  .then(res => res.json())
+ .then((data) => {
+    console.log('Success:', data)
+  })
+  .catch((error) => {
+    console.error('Error:', error)
+  })
+  lasiGarastavokli()
+  uzstadiVaardu(vards+" "+gStavoklis)
```

Comment:
Šeit sanāk, ka pēc garastāvokļa maiņas mums izmainās arī vārds, neliekas pareizi.
Piedāvāju garastāvokli glabāt jaunā mainīgajā un nejaukt to kopā ar vārdu. Jo tagad sanāk, ka pēc katra garastāvokļa maiņas mums nomainās arī vārds...

Reply:
Jā, pie vairākām garastāvokļa maiņām tas vārds izaug ļoti garš, saturot visus iepriekšējos garastāvokļus

Reply:
Tā tam nevajadzētu būt!?
Vārds ir vārds un garastāvoklis ir garastāvoklis.
Ja vajag palīdzību - dod ziņu!